pilot/pkg/config/kube/gateway/templates/deployment.yaml

```diff
@@ -4,8 +4,7 @@ metadata:
   annotations:
     {{ toYamlMap (omit .Annotations "kubectl.kubernetes.io/last-applied-configuration") | nindent 4 }}
   labels:
-    {{ toYamlMap .Labels
-      (strdict "gateway.istio.io/managed" "istio.io-gateway-controller")
+    {{ toYamlMap .DefaultLabels .Labels
```

Comment:
Order matters here. Last one wins; I don't think we want a user to override this?

Reply:
Ah, you're right! Fixing